/etc/gdm/Xsession: Beginning session setup...
Setting IM through im-switch for locale=ca_ES.
Start IM through /etc/X11/xinit/xinput.d/all_ALL linked to /etc/X11/xinit/xinput.d/default.
GNOME_KEYRING_CONTROL=/tmp/keyring-AY2Hwm
SSH_AUTH_SOCK=/tmp/keyring-AY2Hwm/ssh
GNOME_KEYRING_PID=23612
GNOME_KEYRING_CONTROL=/tmp/keyring-AY2Hwm
SSH_AUTH_SOCK=/tmp/keyring-AY2Hwm/ssh
GNOME_KEYRING_CONTROL=/tmp/keyring-AY2Hwm
SSH_AUTH_SOCK=/tmp/keyring-AY2Hwm/ssh
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Backend     : gconf
Integration : true
Profile     : default
Adding plugins
Initializing core options...done
Window created on XQueryTree, map state isViewable? 0
Window created on XQueryTree, map state isViewable? 0
Window created on XQueryTree, map state isViewable? 0
Window created on XQueryTree, map state isViewable? 0
Window created on XQueryTree, map state isViewable? 0
Initializing bailer options...done
Initializing detection options...done
Initializing composite options...done
Initializing opengl options...done
Initializing decor options...done
Initializing mousepoll options...done
Initializing vpswitch options...done
Initializing animation options...done
Initializing snap options...done
Initializing expo options...done
Initializing move options...done
Initializing place options...done
gnome-session[23564]: WARNING: Could not launch application 'screenlet.desktop': Unable to start application: No s'ha pogut executar el procés fill «/home/telecentreuser/screenlet» (El fitxer o directori no existeix)
Initializing grid options...done
Initializing gnomecompat options...done
Initializing wall options...done
Initializing ezoom options...done
Initializing workarounds options...done
Initializing staticswitcher options...done
** (process:23659): DEBUG: zeitgeist-datahub.vala:174: Inserting 3 events
Initializing resize options...done
Initializing fade options...done
Initializing scale options...done
I/O warning : failed to load external entity "/home/telecentreuser/.compiz/session/1027cbfefd4f30c72e131125372718144000000235640031"
Initializing session options...done
Home directory /home/telecentreuser not ours.
Starting unity-window-decorator
Failed to play sound: Not available
Setting Update "run_command_terminal_key"
Setting Update "fullscreen_visual_bell"
21/07/2011 15:08:50 Autoprobing TCP port in (all) network interface
21/07/2011 15:08:50 Listening IPv6://[::]:5900
21/07/2011 15:08:50 Listening IPv4://0.0.0.0:5900
21/07/2011 15:08:50 Autoprobing selected port 5900
21/07/2011 15:08:50 Advertising security type: 'TLS' (18)
21/07/2011 15:08:50 Advertising authentication type: 'VNC Authentication' (2)
21/07/2011 15:08:50 Advertising security type: 'VNC Authentication' (2)
Initializing nautilus-gdu extension

(nautilus:23648): GLib-GObject-CRITICAL **: g_object_unref: assertion `G_IS_OBJECT (object)' failed
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.
Ha fallat la connexió: S'ha refusat la connexió
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
** (process:23659): DEBUG: desktop-launch-listener.vala:118: ran with uri: file:///home/telecentreuser
Nautilus-Share-Message: Called "net usershare info" but it failed: 'net usershare' ha retornat l'error 255: net usershare: cannot open usershare directory /var/lib/samba/usershares. Error S’ha denegat el permís
You do not have permission to create a usershare. Ask your administrator to grant you permissions to create a share.


(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
** (process:23659): DEBUG: zeitgeist-datahub.vala:174: Inserting 1 events
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed

(nautilus:23648): GLib-GObject-CRITICAL **: g_value_get_object: assertion `G_VALUE_HOLDS_OBJECT (value)' failed
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...
Home directory /home/telecentreuser not ours.

** (gnome-settings-daemon:23619): WARNING **: Connection failed, reconnecting...

(polkit-gnome-authentication-agent-1:23658): GLib-GObject-WARNING **: instance of invalid non-instantiatable type `(null)'

(polkit-gnome-authentication-agent-1:23658): GLib-GObject-CRITICAL **: g_signal_handlers_destroy: assertion `G_TYPE_CHECK_INSTANCE (instance)' failed
Shutting down nautilus-gdu extension

--- Hash table keys for warning below:
--> inode/directory
--> l2049
--> root
--> telecentreuser
--> 100

(nautilus:23648): Eel-WARNING **: "unique eel_ref_str" hash table still has 5 elements at quit time (keys above)

(nautilus:23648): Eel-WARNING **: "nautilus-directory.c: directories" hash table still has 6 elements at quit time
gnome-settings-daemon: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
update-notifier: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
gnome-screensaver: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
unity-window-decorator: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.0.
screenlets-daemon.py: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
evolution-alarm-notify: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
no daemon yet
gdu-notification-daemon: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
applet.py: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
nm-applet: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
bluetooth-applet: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
vino-server: Fatal IO error 11 (El recurs no es troba disponible temporalment) on X server :0.
** (process:23659): DEBUG: zeitgeist-datahub.vala:58: Zeitgeist-daemon disappeared from the bus, exitting...

** (process:23659): WARNING **: zeitgeist-datahub.vala:218: Unable to get name "org.gnome.zeitgeist.datahub" on the bus!
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
